### DISCUSSION TITLE
i fond a bug about   monai.transforms.RandSpatialCropd ?

### DISCUSSION THREAD

**Discussion by darlingbud** [Project-MONAI/MONAI#4710] [Q&A]
the scence is 
` RandSpatialCropd(keys=["img", "label", "dist"],  roi_size=(256,256), random_size=False)
`
and  original  `data ={ "img": img, "label" : label, "dist" :dist }`
`img.shape=( 3, 383,512) , label.shape=(1,383,512) , dist.shape=( 32, 383,512)`
after ` data=RandSpatialCropd(data) `operation ,i got the `data`  
`img.shape=( 3, 256,256) , label.shape=(1,208,256) , dist.shape=( 32, 208,512)`

is this a BUG ?  or i just mistake something about the function ?

**Comment by darlingbud** [comment]
sorry， i find the reason ，the imread from different Lib have different result ，sometimes W*H sometimes H*W .
 so the shape is ( 512, 383)  and  ( 383 , 512)
i am sorry about this mistake agine

**Reply by Nic-Ma** [reply]
No problem, welcome to provide feedback!

Thanks.
